content/docs/1.6.0/volumes-and-nodes/storage-class-parms.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Storage Class Settings
+weight: 3
+---
+
+## Overview
+
+Storage Class as a resource object has a number of settable parameters.  See also [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes).  
+Most of the ones in this list are specific to Longhorn's driver.  A few are common to all Kubernetes storage classes:
+
+### Default Class
+Set this Storage Class as the default for dynamic (Kubernetes-driven) volume creation.  
+Type: Boolean  
+Notes: This should only be true for one StorageClass object.  If not, either all volume creations will fail, or else one of the competing defaults will be chosen, and it may be hard to predict which one.  
+***(Which is it?  Give this a try.)***
+
+### Provisioner
+Specifies the plugin that will be used for dynamic creation of persistent volumes.  For Longhorn, that is always "driver.longhorn.io"
+
+### Reclaim Policy
+Define what happens to a persistent volume when released from its claim (PVC), when the claim is deleted, say.  This is strictly a Kubernetes setting.  Longhorn itself does not validate the setting nor make any decisions based on it.  
+Type:  Enumeration, either "Delete" or "Retain".  
+Notes:  This is a two-valued enumeration rather than a Boolean because there was a third setting, "Recycle", which is deprecated and should be avoided.
+
+## Longhorn-specific Parameters
```

Comment:
Should we clarify that the previous settings were appropriate top level, but that these exist within the parameters field?

```
topLevelSetting:
topLevelSetting:
parameters:
  longhornSetting:
```